environments/staging/terraform.yml

```diff
@@ -249,18 +249,20 @@ elasticache:
   create: no
 
 elasticache_cluster:
+  create: yes
   cache_node_type: "cache.t3.medium"
-  node_groups:  2
-  replicas_per_node: 1
   cache_engine_version: "4.0.10"
+  cache_prameter_group: "default.redis4.0"
   automatic_failover: true
   transit_encryption: false
   at_rest_encryption: true
   auto_minor_version: false
+  cluster_size: 2
   maintenance_window: "sun:03:30-sun:04:30"
   snapshot_retention: 7
   snapshot_window: "07:30-08:30"
 
 r53_private_zone:
+  create: yes
   domain_name:  "stg.dimagi.local"
```

Comment:
I'm curious what this is used for and whether we can give it a better name. Since the networks between environments are totally isolated, if this is the endpoint other machines will use to access redis, perhaps we can just call it "redis.local"? Or if it's good to be more explicit, we can call it "redis.staging.local" here (and "redis.production.local", "redis.india.local" on the other environments?)

Something worth getting right before we apply on production, but no need to block this PR

Reply:
we agreed to `*.staging.commcare.local` so in this case `redis.staging.commcare.local`

Reply:
I have updated domain name [ staging.commcare.local ] and added "create_record option" in terraform [environments/staging/terraform.yml] environment file and creates Route-53 record based on create_record key value i.e. yes|no.